src/modules/hyprland/workspaces.cpp

```diff
@@ -43,18 +57,19 @@ Workspaces::Workspaces(const std::string &id, const Bar &bar, const Json::Value
     active_only_ = config_active_only.asBool();
   }
 
-  box_.set_name("workspaces");
-  if (!id.empty()) {
-    box_.get_style_context()->add_class(id);
+  auto config_sort_by = config_["sort-by"];
+  if (config_sort_by.isString()) {
+    sort_by = config_sort_by.asString();
```

Comment:
Convert the string argument to an enum here. Fallback to DEFAULT if given argument is misspelled (with a warning)